### ISSUE TITLE
Pull access denied for flame

### ISSUE THREAD

**Issue by aaronstuder** [pawelmalak/flame#183]
_sudo docker run -p 5005:5005 -v /home/<user>/flame:/app/data -e PASSWORD=flame_password flame
Unable to find image 'flame:latest' locally
docker: Error response from daemon: pull access denied for flame, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'._

_sudo docker pull pawelmalak/flame     
Using default tag: latest
latest: Pulling from pawelmalak/flame
Digest: sha256:ccdfec957cadad538b7fbb4b32647cffb1c8d3f6de93f211a497e5d4121d1b18
Status: Image is up to date for pawelmalak/flame:latest
docker.io/pawelmalak/flame:latest_

I feel like I'm missing something silly here

**Comment by pawelmalak** [comment]
You are pulling image from `pawelmalak/flame` but in your run command you are only passing `flame` as an image name.

You also have double // in your volume path.

**Comment by aaronstuder** [comment]
Oh, I was just following the Installation from the readme. I'm somewhat new with docker. Thanks for the quick reply!

**Comment by pawelmalak** [comment]
That's ok. Did you get it working?

**Comment by aaronstuder** [comment]
Yes. Thanks again! You might want to edit to make a bit more n00b friendly. You could also add a note about using -d so it runs in the background :)
